Profiles - ZIB 2017/nl-core-contactpoint.xml

```diff
@@ -422,6 +444,23 @@
       <short value="TelecomType / NumberType / EmailAddressType" />
       <definition value="Purpose of this contact point" />
       <alias value="TelecomType / NummerSoort / EmailSoort" />
+      <binding>
+        <strength value="required" />
+        <valueSetReference>
+          <extension url="http://hl7.org/fhir/StructureDefinition/11179-permitted-value-conceptmap">
+            <valueReference>
+              <reference value="http://nictiz.nl/fhir/ConceptMap/EmailSoortCodelijst-to-ContactPointUse" />
+              <display value="EmailSoortCodelijst-to-ContactPointUse" />
+            </valueReference>
+          </extension>
+          <extension url="http://hl7.org/fhir/StructureDefinition/11179-permitted-value-conceptmap">
+            <valueReference>
+              <reference value="http://nictiz.nl/fhir/ConceptMap/TelecomTypeCodelijst-to-ContactPointUse" />
+              <display value="TelecomTypeCodelijst-to-ContactPointUse" />
+            </valueReference>
+          </extension>
+        </valueSetReference>
+      </binding>

```

Comment:
The mapping.comment mentioning of the ConceptMaps deviates from other mapping.comments.